### PR TITLE
Path finder ros node 

### PR DESCRIPTION
**Pull request by icecoldfire**
You can now make (catkin_make) this node.

### CODE REVIEW

File: path_finding/package.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<package>
+  <buildtool_depend>catkin</buildtool_depend>
```
Comment (0phoff) [inline path_finding/package.xml]: could you move these lines to the lines 44 and below? That way the dependencies are nicely grouped! 👍

File: path_finding/package.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  <build_depend>message_generation</build_depend>
```
Comment (0phoff) [inline path_finding/package.xml]: could you move the lines 3-4 to the lines 44 and below? That way the dependencies are nicely grouped! 👍

File: path_finding/package.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  <!-- Use build_depend for packages you need at compile time: -->
+  <!--   <build_depend>message_generation</build_depend> -->
+  <!-- Use buildtool_depend for build tool packages: -->
+  <!--   <buildtool_depend>catkin</buildtool_depend> -->
+  <!-- Use run_depend for packages you need at runtime: -->
+  <!--   <run_depend>message_runtime</run_depend> -->
+  <!-- Use test_depend for packages you need only for testing: -->
+  <!--   <test_depend>gtest</test_depend> -->
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>message_generation</build_depend>
```
Comment (0phoff) [inline path_finding/package.xml]: Normally when adding message_generation as build dependency  you should add message_runtime as a run dependency!!
If you compiled without errors, I am not sure you need message_generation? (didnt look at the rest of the code)

Comment (icecoldfire) [inline path_finding/package.xml]: It compiles and the test function works. I use a String message for now.

**Pull request merged**